partition/roles/monitoring/prometheus/README.md

```diff
@@ -0,0 +1,29 @@
+# prometheus
+
+Deploys prometheus in a systemd-managed Docker container.
+
+## Variables
+
+| Name                                             | Mandatory | Description                                                 |
+| ------------------------------------------------ | --------- | ----------------------------------------------------------- |
+| prometheus_port                                  |           | Port for prometheus                                         |
```

Comment:
Missing `prometheus_image_name` and `prometheus_image_tag`.

Reply:
Should we make them `Mandatory`?

Reply:
Yep, completely missed that. It's not defaulted, so it's required.

It will come through the release vector but from the perspective of the role it's required.